includes/forms.inc

```diff
@@ -41,8 +41,32 @@ function islandora_ip_embargo_settings_form($form, &$form_state) {
       0
     ),
   );
+  $form['islandora_ip_embargo_overlay_text'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Embargoed item overlay text'),
+    '#description' => t('Value for overlay text on embargoed items.'),
+    '#default_value' => variable_get(
+      'islandora_ip_embargo_overlay_text',
+      'EMBARGOED'
+    ),
+  );
+  $form['islandora_ip_embargo_overlay_text_color'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Embargoed item overlay text color'),
+    '#description' => t('Value for overlay text color in hexideciamal format (E.g. FF0000).'),
+    '#maxlength' => 6,
+    '#default_value' => variable_get(
+      'islandora_ip_embargo_overlay_text_color',
+      'FF0000'
+    ),
+  );
+  $form['submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Save configuration'),
+    '#submit' => array('islandora_ip_embargo_settings_submit'),
+  );
 
-  return system_settings_form($form);
+  return $form;
```

Comment:
Shouldn't need to add a custom submit as `system_settings_form` should handle the setting of these vars explicitly.